tools/walletextension/rpcapi/ethereum_api.go

```diff
@@ -0,0 +1,57 @@
+package rpcapi
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ten-protocol/go-ten/lib/gethfork/rpc"
+)
+
+type EthereumAPI struct {
+	we *Services
+}
+
+func NewEthereumAPI(we *Services,
+) *EthereumAPI {
+	return &EthereumAPI{we}
+}
+
+func (api *EthereumAPI) GasPrice(ctx context.Context) (*hexutil.Big, error) {
+	return (*hexutil.Big)(big.NewInt(int64(0x3b9aca00))), nil
```

Comment:
This probably needs a comment if it's staying hardcoded. (Might be nice if it was coming from a more central constant too.)

Reply:
it was for debugging. will change back.